src/xdp/program.c

```diff
@@ -1708,15 +1734,17 @@ EbpfProgramOnClientDetach(
     )
 {
     XDP_PROGRAM_OBJECT *ProgramObject = EbpfExtensionClientGetProviderData(DetachingClient);
+    NTSTATUS Status = STATUS_PENDING;
```

Comment:
nit: we are not touching `Status` anyways. why not just write `return STATUS_PENDING`?

Reply:
The `TraceExitStatus` WPP trace at the end of the function implicitly logs whatever variable is named `Status`. It's an awkward situation, but the best possible solution here.

Reply:
(Previously it was `TraceExitSuccess` which directly logs `STATUS_SUCCESS` rather than logging a variable)